src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-customtag-update.js

```diff
@@ -106,6 +106,8 @@ export default class EzBtnCustomTagUpdate extends EzWidgetButton {
      * @return {Object} The rendered select.
      */
     renderSelect(config, attrName) {
+        this.choicesLabel = config.choicesLabel;
```

Comment:
`this.choicesLabel` is available in the whole class and if there will be more attributes with type `select` this will be overridden every time the select occurs. This might be risky.
I would just bind this in the `renderChoice`. 
```js
{config.choices.map(this.renderChoice.bind(this, config.choicesLabel))}
```

Reply:
https://github.com/ezsystems/ezplatform-admin-ui/pull/1001/commits/1a4a78088baf6a58e06c1395c58caa8bcd082e8f
I was not aware that you can add extra arguments in `bind`. Now it is fixed.